tools/images/http_cache/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:bionic
+MAINTAINER peter.ebden@gmail.com
+
```

Comment:
worth adding a VOLUME directive for `/var/www`

Reply:
Dunno if necessary, am mostly using this as a throwaway rather than anything serious

Reply:
OK on further thought have done - doesn't matter much but I suppose it is better (also fixed some permissions issues along the way)